sdk/core/core-arm/package.json

```diff
@@ -79,8 +79,8 @@
     "integration-test:browser": "echo skipped",
     "integration-test:node": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "lint:fix": "eslint -c ../../.eslintrc.old.json src test samples --ext .ts --fix --fix-type [problem,suggestion]",
-    "lint": "eslint -c ../../.eslintrc.old.json src test samples --ext .ts -f html -o core-arm-lintReport.html || exit 0",
+    "lint:fix": "echo skipped",
+    "lint": "echo skipped",
```

Comment:
`eslint` already looks at parent directory for `.eslintrc.json` and it tries to use the one that uses the plugin but this packages does not have the plugin as a dependency and it is hard to add it and clean it up.

Reply:
Since we plan to remove this package altogether (see #11431), am ok with disabling linting here.